mingus/core/scales.py

```diff
@@ -44,11 +45,21 @@
  * MelodicMinor(note)
  * Bachian(note)
  * MinorNeapolitan(note)
+ 
+The bebop scales (new in Sidewinder)
```

Comment:
I see that Sidewinder is a project of yours. These `new in Sidewinder` seem out of context here.